inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -802,9 +805,100 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
             if (parent2->is_type<convolution>() && !conv_supports_fusings(parent2->as<convolution>()))
                 return;
 
-            // This fusing can be extended to support peer node in any layout
-            bool merge_allowed = fused_node->get_users().size() == 1;
+            bool merge_allowed = true;
+            if (fused_node->is_type<convolution>() && fused_node->get_users().size() > 1) {
```

Comment:
only allow to fuse to convolution?

Reply:
I think  we only verify the case as convolution, but I will remove the condition and check FC and DeConv case too.
Thanks